relay-server/src/utils/managed_envelope.rs

```diff
@@ -255,6 +253,15 @@ impl ManagedEnvelope {
         relay_common::instant_to_date_time(self.envelope().meta().start_time())
     }
 
+    /// Set the `done` flag to true.
+    ///
+    /// This allows us to take the envelope out and drop the context without reporting the
+    /// rejection.
```

Comment:
I think this keeps it to the point? Your call.
```suggestion
    /// Remove the envelope and drop the context, without reporting the rejection.
```